base/docker/confd/templates/cert_config.json.tmpl

```diff
@@ -0,0 +1,19 @@
+{
+  "signing": {
+    "default": {
+      "auth_remote": {
+        "remote": "caserver",
+        "auth_key": "key1"
+      }
+    }
+  },
+  "auth_keys": {
+    "key1": {
+      "key": "{{if exists "/opg/base/cakey" }}{{getv "/opg/base/cakey"}}{{end}}",
```

Comment:
Yes and no, you have csr/xxx variables as well, just think it's a tiny bit clearer than cakey :)

Though it is pendantry and I am getting quite fond of cake in code